src/data/faq_de.json

```diff
@@ -160,6 +160,16 @@
                             "in die Corona-Warn-App ein."
                         ]
                     },
+                    {
+                        "title": "Rote Karte? So geht es zum Corona Test",
+                        "anchor": "red_card_how_to_test",
+                        "active": true,
+                        "textblock": [
+                            "Wenn die Corona-Warn-App Ihnen mit der roten Karte ein hohes Infektionsrisiko anzeigt, isolieren Sie sich bitte zügig und konsultieren Sie einen Arzt, um sich ggf. testen zu lassen.",
+                            "Für einen reibungslosen Ablauf beim Testen, zum Empfang des Testbefundes und zu Warnung Anderer bei einem positiven Ergebnis folgen Sie bitte den Schritten in <a href='../../assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ02_x3.pdf' target='_blank' rel='noopener'>dieser Übersicht vom Robert-Koch-Institut</a>.",
+                            "<a href='../../assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ02_x3.pdf' target='_blank' rel='noopener'><img src='../../assets/documents/BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.jpg' alt='PDF'></a>"

```

Comment:
Link to
BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ02_x3.pdf does not work. The document is
BPA_Corona-Warn-App_RKI-Plakat_Testcenter_A1_ICv2_RZ04_x3b.pdf